matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/Event.java

```diff
@@ -93,6 +93,7 @@
     public static final String EVENT_TYPE_ROOM_KEY_REQUEST = "m.room_key_request";
     public static final String EVENT_TYPE_FORWARDED_ROOM_KEY = "m.forwarded_room_key";
     public static final String EVENT_TYPE_URL_PREVIEW = "org.matrix.room.preview_urls";
+    public static final String EVENT_TYPE_SERVER_NOTICE_USAGE_LIMIT = "m.server_notice.usage_limit_reached";
```

Comment:
Is it the best place to add this String? Maybe move it to ServerNoticeUsageLimitContent class?

Reply:
Yes sorry, it's a miss from me, at start it was supposed to be a true event type.